client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.js

```diff
@@ -71,12 +71,52 @@ BreadcrumbLink.defaultProps = {
   isPublic: false
 };
 
+const RootProjectsLink = ({ api, section, operation, label }) => {
+  const dispatch = useDispatch();
+  const onClick = e => {
+    e.stopPropagation();
+    e.preventDefault();
+    dispatch({
+      type: 'DATA_FILES_SET_MODAL_PROPS',
+      payload: {
+        operation,
+        props: { showProjects: true }
+      }
+    });
+  };
+  if (section === 'modal') {
+    return (
+      <span>
+        <a
+          className="breadcrumb-link"
+          href={`/workbench/data/${api}/projects/`}
+          onClick={onClick}
+        >
+          {label}
+        </a>
+      </span>
+    );
+  }
+  return (
+    <Link className="breadcrumb-link" to={`/public-data/${api}/projects/`}>
```

Comment:
This needs to be switched for the non-modal scenario as that is failing. 

 ```suggestion
    <Link className="breadcrumb-link" to={`/workbench/data/${api}/projects/`}>
```